pymdownx/superfences.py

```diff
@@ -556,19 +606,33 @@ def search_nested(self, lines):
                 lines = lines[:start] + [fenced] + lines[end:]
         return lines
 
-    def highlight(self, src, language):
+    def highlight(self, src, language, options=None):
```

Comment:
It's weird to have options passed in but use `self.options`, I should just use the passed in one.